packages/amplify-category-function/src/provider-utils/awscloudformation/utils/functionPluginLoader.ts

```diff
@@ -195,6 +201,7 @@ async function getSelectionsFromContributors<T>(
   if (!Array.isArray(selection)) {
     selection = [selection];
   }
+  selection = _.uniq(selection); // remove duplicates
```

Comment:
how could there be duplicates?

Reply:
This is because earlier I was doing some other implementation.
Thanks for catching this . Fixed